### ISSUE TITLE
More Accessibility things

### ISSUE THREAD

**Issue by wunc** [utdal/profiles#87]
More accessibility suggestions from the latest scans

- [x] Allow users to quickly skip to content
  - skip links / `<main>` landmark: https://webaim.org/techniques/skipnav/
- [ ] Combine adjacent links with the same destination
  - on e.g. Browse all page cards, the image and title both link to the same place. Maybe we remove the title link?
- [x] Ensure every page contains a top-level heading, i.e. `<h1>`
  - /, /faq, /login, any empty profile pages
  - maybe change the profile name from h2 to h1?
- [ ] Ensure links explain their purpose
  - The paginator "next page" and "previous page" only uses icons, not text. Maybe add an aria-label?
- [x] Mark navigation as lists
  - On the tags list page, represent the groups of tags as `<ul>`
- [ ] Ensure form controls have labels
  - In the form fields of the profile tags edit modal
- [x] Fix duplicate IDs
  - On the search results page, the "By Tag" and "By Keyword" headers have the same ID
- [x] Ensure links are distinguished by more than color. Increase contrast
  - On the homepage, "name or keyword..." and the tags should have increased contrast and underline style (or maybe show as tags?)
- [x] Ensure dialogs have labels
  - The profile tag editor modal pop-up should have an aria-label to describe it
- [x] Ensure headings include text
  - On the students/about page, the big arrows are h2. Maybe just use a styled div.h2.my-5 instead, with a aria-role=decoration
- [x] Ensure text has sufficient contrast
  - [x] On the profile cards, the "View Profile" text
  - [x] On the homepage, the tags
  - [x] On the profile page, the person titles
  - [x] On the login page, the login button
- [x] Ensure form controls change appearance when selected
  - [x] The main search box should have an outline when focused
  - [ ] The tags modal fields should have an outline when focused
- [x] Ensure pages don't require zooming and 2D scrolling on small screens
  - Fix the super long tags (and maybe let them wrap?)
- [x] Identify the purpose of fields programmatically
  - [x] On the login page, the password field should have an `autocomplete="current-password"` and the username field should have `autocomplete="username"`
  - [ ] On the profile tags edit modal, some sort of autocomplete setting. Maybe "off" since we provide our own autocomplete.
- [x] Ensure form controls contrast sufficiently with their surroundings
  - form field borders everywhere should have a 3:1 contrast on their borders. Example: the login page form fields

**Comment by wunc** [comment]
Most of these were handled in #142, as well as subsequent commit 9d9a3db4.

There are a few remaining, mostly relating to the tags modal picker, but I'm just going to close this out for now.
